### DISCUSSION TITLE
spring boot & dubbo properties

### DISCUSSION THREAD

**Discussion by HarveyBlocks** [apache/dubbo#14034] [Question] [answered]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

When I integrate Dubbo with SpringBoot, I find it difficult to locate description documentation regarding its configuration. For example, I am unsure how to configure enabling metadata reporting, etc. Could you provide me with some explanatory documentation? Thank you.♥♥♥

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
@chickenlj PTAL

**Comment by chickenlj** [comment, marked as answer]
@HarveyBlocks You can read the following two pages. But please notice that they are not officially released yet, we are still working on them so the content might change in the future.

https://dubbo-next.staged.apache.org/zh-cn/overview/mannual/java-sdk/reference-manual/config/spring/spring-boot/#applicationyaml

https://dubbo-next.staged.apache.org/zh-cn/overview/mannual/java-sdk/reference-manual/config/properties/
